mock-core-configs/etc/mock/templates/centos-stream-8.tpl

```diff
@@ -36,23 +36,23 @@ skip_if_unavailable=False
 
 [baseos]
 name=CentOS Stream $releasever - BaseOS
-mirrorlist=http://mirrorlist.centos.org/?release=$stream&arch=$basearch&repo=BaseOS&infra=$infra
-#baseurl=http://mirror.centos.org/$contentdir/$stream/BaseOS/$basearch/os/
+mirrorlist=http://mirrorlist.centos.org/?release=8-stream&arch=$basearch&repo=BaseOS&infra=$infra
```

Comment:
Instead of `8-stream`, can you use `$releasever-stream`?

Reply:
I can, Done.